src/renderer/views/SubscribedChannels/SubscribedChannels.vue

```diff
@@ -230,10 +265,16 @@ watch(activeSubscriptionList, () => {
   filterChannels()
 })
 
-onMounted(() => {
+;(() => {
   document.addEventListener('keydown', keyboardShortcutHandler)
   getSubscription()
-})
+
+  const oldQuery = route.query.searchQueryText ?? ''
+  if (oldQuery !== null && oldQuery !== '') {
+    // `handleQueryChange` must be called after `filterHistoryDebounce` assigned
+    handleQueryChange(oldQuery, true)
+  }
+})()
```

Comment:
Could you please explain why you changed this from an onMounted callback to an immediately invoked function?

Reply:
It was changed from `mounted` to `created` but in composition style either it's in setup block
However `oldQuery` is only used inside `created` so I wrap it with an immediately invoked function

Reply:
The keyboard shortcut handler should be moved back into `onMounted` as we shouldn't be intercepting key presses before the user can actually see the component. As for the other stuff I don't think it's worth making it more complicated with an immediately invoked function because setup is compiled to a function, so the variable will get garbage collected as soon as it reaches the bottom of it anyway.